modules/ml/include/opencv2/ml.hpp

```diff
@@ -1496,6 +1496,187 @@ class CV_EXPORTS_W LogisticRegression : public StatModel
     CV_WRAP static Ptr<LogisticRegression> create();
 };
 
+
+/****************************************************************************************\
+*                        Stochastic Gradient Descent SVM Classifier                      *
+\****************************************************************************************/
+
+/*!
+@brief Stochastic Gradient Descent SVM classifier
+
+SVMSGD provides a fast and easy-to-use implementation of the SVM classifier using the Stochastic Gradient Descent approach,
+as presented in @cite bottou2010large.
+
+The classifier has 5 parameters. These are
```

Comment:
I count 6 items.
What is about something like this: `The classifier has these parameters:`?